MagicPi/actions/WatcherAction.py

```diff
@@ -0,0 +1,31 @@
+import pathos.multiprocessing as mp
+import os;
+import time
+
+class WatcherAction:
+
+    environement = None;
+
+    def __init__(self, Environement):
+        self.environement = Environement;
+
+    def run(self):
+        pool = mp.ProcessingPool(nodes=1)
+        pool.amap(WatcherAction.asynchWatcher,[self])
+        print "Watcher en cours d'execution"
```

Comment:
j'aurai juste fait ce print pour rester cohérent :
`print "[*] Watcher is running...`